cleverhans/picklable_model.py

```diff
@@ -354,3 +354,76 @@ def fprop(self, x):
 
     def get_params(self):
         return []
+
+
+class Add(Layer):
+    """
+    A Layer that adds a function to its input.
+    The function to add is specified in terms of multiple layers, just like
+    in the MLP class.
```

Comment:
This comment mentions multiple layers but the one in `fprop` says only one parent is allowed. Could you explain why?

Reply:
This comment means the Add layer itself contains several layers. The output of those internal layers is added to the one layer that is the parent of the Add layer. For example, to use a single resnet layer, you use an Add(layers=[Conv2D, ReLU, Conv2D]).

Reply:
Got it, thanks for clarifying. I had missed that the comment applied to layers in `self.layers` rather than the parent they are being added to.